### PR TITLE
fix: use package.json language scopes

### PR DESCRIPTION
**Pull request by blakedietz**
Separate out by language type in package.json instead of only using the scope field

### CODE REVIEW

**Comment by blakedietz** [comment]
@aifrak tossing out a potential solution. I checked in the binary for sake of simplicity for testing on your behalf.

If I could get another set of eyes from you to make sure this fixes the problems you were seeing, that would be great:

- Go to vscode
- Uninstall the extension
- Open the command palatte
- Type install from visx
- Select the visx

Go through some of the scenarios you reported.

If you don't want to use the included visx and want to build it yourself, check out the README for how to build.

File: lib/app.ex [hunk 1]
```diff
@@ -115,7 +113,20 @@ defmodule App do
 
     File.write("./README.md", docs)
 
-    File.write!("./snippets.json", snippet_json)
+    elixir_snippet_json =
```
Comment (blakedietz) [inline lib/app.ex:116]: Will make this a bit nicer once we figure out if this solves the problem of scoping to the root.

**Comment by aifrak** [comment]
@blakedietz I have tested with the provided visx and suggestions are working correctly, even for LiveView tags in LiveView templates or in ~H sigil.

**Comment by blakedietz** [comment]
> @blakedietz I have tested with the provided visx and suggestions are working correctly, even for LiveView tags in LiveView templates or in ~H sigil.

Thank you for the speedy reply. I'll remove the visx and tidy up the changes. :)

**Pull request merged**